static/src/javascripts/projects/common/modules/discussion/comment-box.js

```diff
@@ -145,6 +146,25 @@ CommentBox.prototype.prerender = function() {
     }
 };
 
+CommentBox.prototype.showOnboarding = function(e) {
+    e.preventDefault();
+
+    // Check if new commenter as they may have already commented on this article
+    if (this.hasState('onboarding-visible') || !this.options.newCommenter) {
```

Comment:
@OliverJAsh the imagined scenario is a user that comments once (for the first time) and then comments again without reloading the page.